src/main/resources/migration/V20170208000000__add_parent_account.sql

```diff
@@ -0,0 +1,3 @@
+alter table analytics_accounts add parent_account_id varchar(36) default null after account_external_key;
+alter table analytics_accounts add parent_account_name varchar(100) default null after parent_account_id;
+alter table analytics_accounts add parent_account_external_key varchar(50) default null after parent_account_name;
```

Comment:
I know we still use 50 elsewhere (e.g. `account_external_key`), but better to make it 255 instead so it matches the main DDL (we should also probably migrate other columns at some point...).

See https://github.com/killbill/killbill/issues/604.